src/pipgrip/pipper.py

```diff
@@ -157,11 +158,18 @@ def _get_wheel_args(index_url, extra_index_url, pre, cache_dir=None):
 
 
 def install_packages(
-    packages, index_url, extra_index_url, pre, cache_dir, editable, constraints=None
+    packages,
+    index_url,
+    extra_index_url,
+    pre,
+    cache_dir,
+    editable,
+    constraints=None,
+    user=None,
```

Comment:
could you put user above constraints without default? (required positional argument)

install_packages is i think only called once, from cli.py, and should always pass the boolean (default False, maybe after [`--editable`](https://github.com/ddelange/pipgrip/blob/0.3.0/src/pipgrip/cli.py#L140)) and propagate it from there.

then afterwards `pipgrip -h` will give you updated output you can copypaste into the README :)

Reply:
a simple sanity check like for `editable` would also be nice: https://github.com/ddelange/pipgrip/blob/0.3.0/src/pipgrip/cli.py#L268-L270

Reply:
Thanks -- I think it's all done; perhaps you'll want to tweak the order of the arguments in `cli.py` or similar.

Reply:
Done! Now everywhere the order is install -- editable -- user. Also I kept `--user` only, to stay as close to the pip [interface](https://pip.pypa.io/en/stable/reference/pip_install/#usage) as possible (like with editable).